plugins/trace_api_plugin/store_provider.cpp

```diff
@@ -307,8 +307,11 @@ namespace eosio::trace_api {
                _maintenance_condition.wait(lock);
             }
 
+            if (_maintenance_shutdown) {
+               break;
+            }
+
```

Comment:
Should `break` be done after `lock.unlock();` below?

Reply:
The destructor of `unique_lock` will unlock the lock.